src/config.ts

```diff
@@ -32,6 +32,8 @@ export interface Config {
     /** The Address of the Registry contract */
     registry?: HexString;
   };
+  /** CurrentUserId stores the id of the currently authenticated user */
+  currentUserId?: string;
```

Comment:
Currently authenticated, what does this mean? I thought we were moving away from User anyway.  Isn't this always going to be a DSNP Id?

Reply:
My thinking here was that passing a `fromId` to every single call will be inconvenient for clients, so we could store and fetch the DSNP Id for the current user in the config, and server implementers handling calls for multiple users could pass the id as a `ConfigOpt` param to the same calls.

I'll rename it to `currentFromId` for more clarity and to get rid of the user reference.